actions-cleanup-openstack-vms/action.yaml

```diff
@@ -0,0 +1,85 @@
+name: 'Clean up OpenStack vms'
+description: 'this action is to clean up vms in openstack'
+
+
+inputs:
+  DELTA:
+    description: the default age of the VM
```

Comment:
See below, we need to be clear if we talk about running time or age from creation date.